scss-partials/_general.scss

```diff
@@ -0,0 +1,265 @@
+    width: 100%;
+    position: relative;
+    height: 48px;
+    font-family: $font-regular;
+
+    @include media ( ">=md" ) {
+        height: 56px;
+    }
+
+    @include media ( ">=lg" ) {
+        line-height: 24px;
+        height: 61px;
+    }
+
+    &__column {
+        width: 33.33%;
+    }    
+}
+
+.logo {
+    text-align: center;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+
+    &__bow {
+        height: 10px;
+        width: 19px;
+        margin: 12px 0px 0px;
+    
+        @include media ( ">=lg" ) {
+            height: 19px;
+            width: 37px;
+            margin: 16px 0px 0px;
+        }
+    }
+    
+    &__name {
+        width: 117px;
+        height: 6.5px;
+        margin: 7px 0px 0px;
+    
+        @include media ( ">=lg" ) {
+            width: 220px;
+            height: 12px;
+            margin: 13px 0px 0px;
+        }
+    }
+}
+
+.control {
+    text-align-last: right;
+
+    &__option {
+        text-align: center;
+        position: relative;
+        display: inline-block;
+        cursor: pointer;
+        height: 20px;
+        text-align: center;
+        margin: 14px 56px 0px 0px;
+    
+        @include media ( ">=md" ) {
+            margin: 16px 56px 0px 0px;
+        }
+    
+        @include media ( ">=lg" ) {
+            margin: 24px 16px 0px 0px;
+            height: 24px;
+        }
+    }
+}
+
+.link {
+    cursor: pointer;
+
+    &:hover { text-decoration: underline; }
+}
+
+.create {
+    background: $tws_blue;
+    color: $tws_white;
+    text-transform: uppercase;
+    width: 94px;
+    padding: 4px 16px;
+    margin-right: 32px;
+
+    &:hover { background: $tws_coral; }
+
+    &__list {
+        text-align: center;
+        font-weight: bold;
+        letter-spacing: 0.4px;
+    }
+}
+
+.create, .findCouple {
+    display: none;
+
+    @include media ( ">=lg" ) {
+        display: inline-block;
+    }
+}
+
+.menu {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0;
+    font-family: $font-regular;
+
+    @include media ( "<lg" ) {
+        display: none;
+    }
+
+    &__list {
+        display: flex;
+        justify-content: space-evenly;
+        width: 100%;
+        padding: 0px;
+        margin: 0px;
+    
+        &--item {
+            margin: 32px 8px 0px;
+            padding: 0px 0px 8px;
+            color: $tws_black;
+            border-bottom: 2px solid $tws_white;
+            letter-spacing: 0.8px;
+            white-space: nowrap;
+            list-style: none;
+            line-height: 20px;
+            cursor: pointer;
+    
+            &:hover { border-bottom: 2px solid $tws_coral; }
+        }
+    }
+}
+
+.search__img {
+    cursor: pointer;
+    height: 15px;
+    width: 15px;
+    margin: 17px 0px 7px 17px;
+
+    @include media ( ">=md" ) {
+        margin: 21px 0px 7px 17px;
+    }
+    
+    @include media ( ">=lg" ) {
+        margin: 33px 0px 16px 33px;       
+    }
+}
+
+.cta {
```

Comment:
you could separate those styles to a new partial file

Reply:
Ok have created a new service scss file and moved those over